app/src/main/java/com/example/playlistmaker/TrackViewHolder.kt

```diff
@@ -1,30 +1,29 @@
 package com.example.playlistmaker
 
 import android.content.Context
-import android.os.Build
 import android.util.TypedValue
 import android.view.View
 import android.widget.ImageView
 import android.widget.TextView
-import androidx.annotation.RequiresApi
 import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners
 import com.bumptech.glide.request.RequestOptions
 import java.text.SimpleDateFormat
 import java.util.Locale
 
+
 class TrackViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
     private val trackNameView: TextView = itemView.findViewById(R.id.trackName)
     private val trackArtistView: TextView = itemView.findViewById(R.id.trackArtist)
     private val trackTimeView: TextView = itemView.findViewById(R.id.trackTime)
     private val trackImage: ImageView = itemView.findViewById(R.id.trackImage)
 
-    @RequiresApi(Build.VERSION_CODES.S)
     fun bind(model: Track, listener: TrackAdapter.TrackListener) {
 
         val cornerRadiusDp = 2
+//        val cornerRadiusPx = context.getDimensionPixelSize(cornerRadiusDp)
```

Comment:
🍏 Чтобы этот метод заработал, нужно взять радиус из ресурсов
```suggestion
        val cornerRadiusPx = context.getDimensionPixelSize(R.dimen.radius)
```